shared/chat/inbox/container/remote.js

```diff
@@ -23,7 +23,30 @@ const valuesCached = memoize((...vals) => vals.map(v => v))
 const metaMapToFirstValues = memoize(metaMap =>
   metaMap
     .partialSort(maxShownConversations, (a, b) => b.timestamp - a.timestamp)
-    .filter((_, id) => Constants.isValidConversationIDKey(id))
+    .filter((meta, id) => {
```

Comment:
The filter needs to be moved above the `partialSort`.

Reply:
Okay, let me do that.